src/nbl/video/COpenGLSpecializedShader.h

```diff
@@ -51,7 +51,7 @@ class COpenGLSpecializedShader : public core::impl::ResolveAlignment<IGPUSpecial
 				return true;
 			if (!pc.info.name.size()) // cannot handle anonymous push constant blocks (we loose the names)
 			{
-				os::Printer::log("Push Constant blocks need to be named (limitation of SPIR-V Cross). Creation of COpenGLSpecializedShader failed.", ELL_ERROR);
+				m_logger->log("Push Constant blocks need to be named (limitation of SPIR-V Cross). Creation of COpenGLSpecializedShader failed.", system::ILogger::ELL_ERROR);
```

Comment:
this method is invoked as a consequence of a commandbuffer submitting, when you have the log as a member of the COpenGLSpecializedShader, the message is piped to the shader's log to rather than to the log passed to this method from the commandbuffer submission processing thread